### DISCUSSION TITLE
Latest News structure

### DISCUSSION THREAD

**Discussion by florian-duffe** [CommandPost/FCPCafe#203] [FCP Cafe Questions] [answered]
Would it be possible to re-design "latest news" in this way, that every item is a stand-alone-post, so that it could be 
a) posted in Social Media, atm if you post a news item and a few days go by the link leads to the top of the page.
b) discussion per post is possible. atm, if you click on "discuss this news item" you land on the discussion-board without knowing what to do.

c) I'm assuming it would be neccessary to also implement collapsing news items and "read more..." link as well as sort of an archive for the news. - maybe too farfetched.

**Comment by latenitefilms** [comment]
Good question! What I MIGHT be able to do is make the date a link for each news item, which then opens it in its own page.

**Reply by latenitefilms** [reply]
I'm now generating the individual news items, for example: https://fcp.cafe/news/20230618/

Will update the Latest News feed and RSS feed later today, as this will take some experimentation.

Stay tuned.

**Reply by latenitefilms** [reply, marked as answer]
This has now been actioned! Great suggestion!
